test/cli/dune

```diff
@@ -0,0 +1,10 @@
+(include dune.inc)
```

Comment:
dune file generation is a bit heavy for these kind of tests, I'll try to refactor this in a future PR.

Reply:
You mean it's slowing dune ? `dune runtest` already take a few seconds.

Reply:
I mean that it's a bit heavy to have arguments in a file, output in another file, etc. I'll try to see if that's better with a different kind of test, but this works in the meantime.

Reply:
I see. `gen.ml` could read a sexp file, what do you think ?

Reply:
let me work something out in another PR, the most important part is that we now have tests!